src/Castle.Core/Core/Logging/ConsoleLogger.cs

```diff
@@ -32,7 +32,7 @@ public class ConsoleLogger : LevelFilteredLogger
 		///   set to <c>LoggerLevel.Debug</c> and the <c>Name</c>
 		///   set to <c>String.Empty</c>.
 		/// </summary>
-		public ConsoleLogger() : this(String.Empty, LoggerLevel.Debug)
+		public ConsoleLogger() : this(String.Empty, LoggerLevel.Trace)
```

Comment:
Not convinced the console logger should default to trace logging.

Reply:
I reversed this change.